hack/generate-manifests.sh

```diff
@@ -26,20 +26,24 @@
 set -euo pipefail
```

Comment:
Do we wan't to use our custom script to generate manifests? Operator-sdk has `generate k8s` (or something similar). That would generate CRDs with proper OpenAPI checking and more (RBAC requirements).